### PR TITLE
COMMERCE-12776

### PR DESCRIPTION
**Pull request by alessiorendina**
Based on #4496 

https://liferay.atlassian.net/browse/COMMERCE-12776

@andrea-ale-sbarra

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-type-virtual-order-content-web/src/main/resources/META-INF/resources/js/OpenTermsOfUseModalPropsTransformer.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	return {
+		...props,
+		onClick() {
+			openModal({
+				buttons: [
+					{
+						autoFocus: true,
+						displayType: 'secondary',
+						label: Liferay.Language.get('i-disagree'),
+						onClick: ({processClose}) => {
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-type-virtual-order-content-web/src/main/resources/META-INF/resources/js/OpenTermsOfUseModalPropsTransformer.js:28]: this onClick looks not necessary

Comment (alessiorendina) [reply modules/apps/commerce/commerce-product-type-virtual-order-content-web/src/main/resources/META-INF/resources/js/OpenTermsOfUseModalPropsTransformer.js:28]: Right, thanks

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:close

**Pull request closed without merging**